muhireheir/mybrand-1
# setup folder structure

> #### What does this PR do?
> * Establish folderstructure 
> #### Description of Task to be completed
> * Establish folderstructure expecially for  frontend
> #### How should this be manually tested?
>  * clone the repository
> * change to ch-setup branch
> * go into ui folder
> * check if you see the  following subfolders(inside ui folder)
	* css
	* img
	* js
	* pages
> *  check if you see index.html file
> * Done.

> #### Any background context you want to provide?
> * to establish proper folder stucture for the project
> #### What are the relevant Trello story?
>  * Setup
> #### Screenshots (if appropriate)
>
